esi_leap/common/keystone.py

```diff
@@ -73,3 +73,11 @@ def get_project_name(project_id, project_list=None):
         return project.name if project else ''
     else:
         return ''
+
+
+def get_project_email(project_id):
+    if project_id:
+        project = get_keystone_client().projects.get(project_id)
+        return project.email if project else ''
+    else:
+        return ''
```

Comment:
It may make sense to make the default 'None' instead of an empty string

Reply:
Fixed